processing/BoxPlotter.py

```diff
@@ -0,0 +1,36 @@
+from argparse import Namespace
+import matplotlib.pyplot as plt
+from typing import List
+from Result import Key, MetricType, Result
+
+
+class BoxPlotter():
+    def __init__(self, result: Result):
+        self.result: Result = result
+
+    def __select(self, condition: List[str], otherwise: List[str]):
+        return condition if condition else otherwise
```

Comment:
Hvad gør select?

Reply:
Den returnere det første element hvis det er truthy ellers så tager den det andet element.
Den er udelukkende lavet for at reducere mængden af kode. Koden nedenfor er ækvivalent

```python
if condition:
    return condition
else:
    return otherwise
```